src/components/Popover/Popover.stories.tsx

```diff
@@ -49,20 +49,6 @@ export const Default: StoryObj<PopoverProps> = {
   },
 };
 
-export const Arrow: StoryObj<PopoverProps> = {
-  args: {
-    children: (
-      <>
-        <Popover.Button as={Button}>Open Popover</Popover.Button>
-        <Popover.Content showArrow>
```

Comment:
Should the `showArrow` prop be removed as well?

Reply:
Honestly, I think it would be ok to axe the `showArrow` prop and the couple usages in the recipes, since it's not being used downstream